cdk/apps/dev_worker/README.md

```diff
@@ -5,15 +5,25 @@ Deploy an ec2 instance with docker ready to go and logged into our ECR (Elastic
 
 This repo is (partially) based on the cdk example provided [here](https://github.com/aws-samples/aws-cdk-examples/blob/master/python/existing-vpc-new-ec2-ebs-userdata/cdk_vpc_ec2)
 
+## Quick start:
+1. Create the venv
+`python3 -m venv .env`
+2. Activate the venv
+`source .env/bin/activate`
+3. Deploy the ec2 instance:
+`cdk deploy`
+4. Log into the instance:
+*You will need to read below to correctly configure your ssh config.*
+`ssh i-....`
```

Comment:
I would prefer putting SSM instead of SSH in the quickstart.

Reply:
Will this allow rsyncing or port binding? I guess we could also write an alias for port binding too with something from here: https://aws.amazon.com/blogs/aws/new-port-forwarding-using-aws-system-manager-sessions-manager/

Reply:
The rsyncing is just the iap configuration file I like to throw in so I can download and run things from gds on the volume. Not necessary and still possible, ssh isn't ruled out.

Reply:
Yes, SSH is still in place, but I would like users to start getting more acquainted with SSM since it's easier for us to handle from a security/threat perspective: much better integrated with AWS.

So yes, I would keep the SSH notes down below yet indicate that SSM is preferred though (and leave SSM as the canonical way on the quickstart since most users will follow those 5 steps).

Reply:
I don't really see the difference. The ssh is just using ssm underneath. Are we adding in vulnerabilities with the key pairs?

Reply:
No, look at this: https://docs.aws.amazon.com/systems-manager/latest/userguide/session-manager-logging-auditing.html.

If there's a breach we can track it down and evaluate the extent of the damage, with SSH you'd have to instrument quite a few more things (logservers, etc...).

Reply:
So the ProxyCommand in the ssh config is doing something more magical than just a subprocess?

Reply:
Sorry, I see what you mean and I understand now, both options are fine.